webpack/assets/javascripts/react_app/components/common/forms/Select.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import CommonForm from './CommonForm';
+import $ from 'jquery';
+import {map} from 'lodash';
+
+class Select extends React.Component {
+  componentDidMount() {
+    if ($.fn.select2) {
+      $(this.refs.select)
+        .select2()
+        .on('change', this.props.onChange);
+    }
+  }
+
+  render() {
+    const renderOptions = (arr) =>
+			map(arr, (attribute, value) => {
+      return <option key={attribute} value={value}>{value}</option>;
+    });
+
+    const {
+      label,
+      className = '',
+      value,
+      onChange,
+      options,
+      disabled
+    } = this.props;
+
+    const innerSelect = (
+      <select disabled={disabled}
+              ref="select"
+              className="form-control" value={value} onChange={onChange}>
+        <option value="">{__('Please select')}</option>
```

Comment:
We are using select2 for everything, please see the componentDidMount on top of this file.
select2 needs a select element to work on...